### PR TITLE
feat: Norwegian language support

### PR DESCRIPTION
**Pull request by olagrasli**
### Pull Request Checklist

- [x] 1. I have [**rebased**][rebase] the latest version of the master branch into my feature branch and all conflicts
         have been resolved.
- [x] 2. I have added information about the change/addition to functionality to the CHANGELOG.md file under the
         [Unreleased] heading.
- [ ] 3. I have documented my code using JDocs tags.
- [x] 4. I have removed unnecessary commented out code, imports and System.out.println statements.
- [ ] 5. I have written JUnit tests for any new methods/classes and ensured that they pass.
- [ ] 6. I have created API tests for any new functionality exposed to the API.
- [ ] 7. If changes/additions are made to the ors-config.json file, I have added these to the [ors config documentation][config]
         along with a short description of what it is for, and documented this in the Pull Request (below).
- [ ] 8. I have built graphs with my code of the Heidelberg.osm.gz file and run the api-tests with all test passing
- [ ] 9. I have referenced the Issue Number in the Pull Request (if the changes were from an issue).
- [ ] 10. For new features or changes involving building of graphs, I have tested on a larger dataset
          (at least Germany), and the graphs build without problems (i.e. no out-of-memory errors).
- [ ] 11. For new features or changes involving the graphbuilding process (i.e. changing encoders, updating the
          importer etc.), I have generated longer distance routes for the affected profiles with different options
          (avoid features, max weight etc.) and compared these with the routes of the same parameters and start/end
          points generated from the current live ORS.
          If there are differences then the reasoning for these **MUST** be documented in the pull request.
- [x] 12. I have written in the Pull Request information about the changes made including their intended usage
          and why the change was needed.
- [ ] 13. For changes touching the API documentation, I have tested that the API playground [renders correctly][api].

Fixes # .

### Information about the changes
- Key functionality added: Norwegian language
- Reason for change: Extend with Norwegian language

### Examples and reasons for differences between live ORS routes, and those generated from this pull request
-

### Required changes to ors config (if applicable)
-

[config]: https://GIScience.github.io/openrouteservice/installation/Configuration.html
[api]: https://gitlab.gistools.geog.uni-heidelberg.de/giscience/openrouteservice-infrastructure/ors-docs-api#test-new-ors-documentation
[rebase]: https://github.com/GIScience/openrouteservice/blob/master/CONTRIBUTE.md#pull-request-guidelines

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ RELEASING:
 - info on duration format in parameter description ([#1504](https://github.com/GIScience/openrouteservice/pull/1504))
 - YML configuration ([#1506](https://github.com/GIScience/openrouteservice/pull/1506))
 - new backend documentation using VitePress([#1617](https://github.com/GIScience/openrouteservice/pull/1617))
+- support for Norwegian language
```
Comment (koebi) [inline CHANGELOG.md]: Please link the corresponding PR here :)

File: ors-engine/src/main/java/org/heigit/ors/routing/APIEnums.java [hunk 2]
```diff
@@ -420,6 +420,8 @@ public enum Languages {
         NE_NP("ne-np"),
         NL("nl"),
         NL_NL("nl-nl"),
+        NO("no"),
+        NO_NB("no-nb"),
```
Comment (koebi) [inline ors-engine/src/main/java/org/heigit/ors/routing/APIEnums.java]: Hey, I am not sure what is correct, but according to https://www.localeplanet.com/icu/nb-NO/index.html, `nb-no` would be used here?

**Review by koebi — CHANGES_REQUESTED** [review]
Hey,

thank you very much for your contribution :))

As far as I can tell, this looks very good! I've added two small comments, once they are clear we're ready to merge :)

Best regards

File: CHANGELOG.md (hunk 1 above)
Comment (olagrasli) [reply CHANGELOG.md]: I have now updated the changelog with link to the pull request

File: ors-engine/src/main/java/org/heigit/ors/routing/APIEnums.java (hunk 2 above)
Comment (olagrasli) [reply ors-engine/src/main/java/org/heigit/ors/routing/APIEnums.java]: Looks like that is the most common. I have updated it in the file and renamed the file with the translation to match this.

**Review by koebi — APPROVED** [review]
Thanks a lot for the quick response and thank you again for the translation :)
Looks good to me :+1:

**Comment by olagrasli** [comment]
Thank you very much for the warm welcome to the project!
I really enjoy the project.
Looking forward to using it in the future.

**Pull request merged**